content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Getting Started with Session Replay
+kind: documentation
+further_reading:
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/session_replay/'
+      tag: 'Documentation'
+      text: 'Session Replay'
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/session_replay/mobile/'
+      tag: 'Documentation'
```

Comment:
For these docs links + the Troubleshooting link below, you can leave them as `/real_user_monitoring/session_replay/`, but for links outside of the documentation repo, they have to be absolute (full) URLs!